SPECS/gd/gd.spec

```diff
@@ -0,0 +1,636 @@
+%define prjname libgd
+%define lname libgd3
+Summary:        A Drawing Library for Programs That Use PNG and JPEG Output
+Name:           gd
+Version:        2.3.0
+Release:        2.3%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libgd.github.io/
+Source0:        https://github.com/libgd/libgd/releases/download/%{name}-%{version}/%{prjname}-%{version}.tar.xz
+Source1:        baselibs.conf
+# might be upstreamed, but could be suse specific also (/usr/share/fonts/Type1 font dir)
+Patch1:         gd-fontpath.patch
+# could be upstreamed, but not in this form (need ac check for attribute format printf, etc.)
+Patch2:         gd-format.patch
+# could be upstreamed
+Patch3:         gd-aliasing.patch
+BuildRequires:  fontconfig-devel
+BuildRequires:  freetype > 2
+# needed for tests
+BuildRequires:  libjpeg-turbo-devel
+BuildRequires:  libpng-devel
+BuildRequires:  libtiff > 4
+BuildRequires:  libwebp-devel
+BuildRequires:  pkg-config
+Provides:       gdlib = %{version}
+Obsoletes:      gdlib < %{version}
+
+%description
+Gd allows your code to quickly draw images complete with lines, arcs,
+text, and multiple colors. It supports cut and paste from other images
+and flood fills. It outputs PNG, JPEG, and WBMP (for wireless devices)
+and is supported by PHP.
+
+%package -n %{lname}
+Summary:        A Drawing Library for Programs That Use PNG and JPEG Output
+
+%description -n %{lname}
+Gd allows your code to quickly draw images complete with lines, arcs,
+text, and multiple colors. It supports cut and paste from other images
+and flood fills. It outputs PNG, JPEG, and WBMP (for wireless devices)
+and is supported by PHP.
+
+%package devel
+Summary:        Drawing Library for Programs with PNG and JPEG Output
+Requires:       %{lname} = %{version}
+Requires:       glibc-devel
+
+%description devel
+gd allows code to quickly draw images complete with lines, arcs, text,
+multiple colors, cut and paste from other images, and flood fills. gd
+writes out the result as a PNG or JPEG file. This is particularly
+useful in World Wide Web applications, where PNG and JPEG are two of
+the formats accepted for inline images by most browsers.
+
+%prep
+%setup -q -n %{prjname}-%{version}
+%patch1
+%patch2
+%patch3
+chmod 644 COPYING
+
+%build
+# ADDITIONAL CFLAGS ARE NEEDED TO FIX TEST FAILURES IN CASE OF i586, BUT HARMLESS TO APPLY GENERALLY FOR ALL ix86
+%ifarch %{ix86}
+export CFLAGS="%{optflags} -msse -mfpmath=sse"
+%else
+%ifnarch x86_64
+export CFLAGS="%{optflags} -ffp-contract=off"
```

Comment:
This will overwrite our default `CFLAGS`. We're trying to avoid that, if possible (`CFLAGS` may not necessarily equal `%{optflags}`). Can you change that to appending `-ffp-contract=off` to existing `CFLAGS` instead? Similar comment for the line above.